### ISSUE TITLE
Reproducing the results

### ISSUE THREAD

**Issue by SrijithBalachander** [seongjunyun/Graph_Transformer_Networks#25]
Hi authors,

I tried running the code to reproduce the results and I have quite a few questions. I am assuming the number of epochs you trained on is the same as the default (40) in the implementation. The Macro F1 score was fluctuating a bit. Can you tell me the number of times you repeated the experiments? Did you take an average over all the repetitions or considered the maximum? I could only reach the reported value a couple of times out of like 10 repetitions of the code. Eg., for ACM the values ranged from 91.4 to 93.1. But mostly stays around 92.3. Also, you are printing the test score for each epoch. Did  you choose the maximum of that test score or did you test with the model that gave max validation score?

Thanks!

**Comment by seongjunyun** [comment]
Hi SrijithBalachander,

Thank you for your interest in our paper. The Macro F1 score in our paper is the average of the best Macro F1 scores from 10 experiments. As you can see from the code, 
if val_f1 > best_val_f1:
                best_val_loss = val_loss.detach().cpu().numpy()
                best_test_loss = test_loss.detach().cpu().numpy()
                best_train_loss = loss.detach().cpu().numpy()
                best_train_f1 = train_f1
                best_val_f1 = val_f1
                best_test_f1 = test_f1 
,of course, the best Macro F1 score was measured based on the validation set. The performance can change little by little depending on the random seed. 

Thanks!
